packages/webapp/src/containers/Crop/RepeatedCropPlanSpotlight/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { showedSpotlightSelector } from '../../showedSpotlightSlice';
+import { setSpotlightToShown } from '../../Map/saga';
+import React from 'react';
+import { TourProviderWrapper } from '../../../components/TourProviderWrapper/TourProviderWrapper';
+
+export default function RepeatedCropPlanSpotlight({ repeatingPlanCreated, children }) {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const { repeat_management_plan_creation } = useSelector(showedSpotlightSelector);
+  const onFinish = () => dispatch(setSpotlightToShown('repeat_management_plan_creation'));
+
+  return (
+    <TourProviderWrapper
+      open={repeatingPlanCreated && !repeat_management_plan_creation}
+      steps={[
+        {
+          title: t('MANAGEMENT_PLAN.REPEATED_MP_SPOTLIGHT.TITLE'),
+          contents: [t('MANAGEMENT_PLAN.REPEATED_MP_SPOTLIGHT.BODY')],
+          selector: `#repeatPlan0`,
```

Comment:
Off by one, there is no `#repeatPlan0`, the first one is `#repeatPlan1`

Reply:
Fixed with just labeling if it's a repeating plan.